Admin/AdminDependency.php

```diff
@@ -305,6 +305,7 @@ protected function displayStatusLevel($status_level)
 	{
 		$count = $this->getStatusLevelCount($status_level);
 		$first = true;
+		/* @phpstan-ignore-next-line */
```

Comment:
Without it, we get `Access to an undefined property AdminDependency::$entries`.

I added it to suppress this as I did the first pass of changes.  For the other PRs, I just ignored them in the results from phpstan ... so it probably makes sense to remove this comment, just so that if/when we get around to trying to fix all the issues with the code, this one isn't hidden from future phpstan analyses.